collect_app/src/main/java/org/odk/collect/android/audio/AudioRecordingControllerFragment.java

```diff
@@ -105,5 +106,33 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         });
 
         binding.stopRecording.setOnClickListener(v -> audioRecorder.stop());
+
+        formEntryViewModel.isFormControllerSet().observe(getViewLifecycleOwner(), isSet -> {
```

Comment:
It doesn’t feel great to set up an observer for something we know will only be set once but I can’t think of another way to change the view once the form controller is set.